target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -238,6 +238,7 @@ get_status_led() {
 		status_led="$board:amber:power"
 		;;
 	whr-g300n|\
+	zbt-we2026|\
```

Comment:
There are following LEDs on the device:
   power/WLAN/WAN/LAN1/LAN2/LAN3/LAN4

(IDs are taken from http://donar.messe.de/exhibitor/cebit/2017/L364619/zbt-we2026-specification-eng-428415.pdf, I didn't find better document)

The power LED is red.
Other LEDs are green.
I assume the power LED is only red. It is red during bootloader process too.
I haven't disassembled the device.